CONTRIBUTING.md

```diff
@@ -44,22 +44,18 @@ The extension has three major modules, which are listed as follow:
 3. Install the node dependencies: `npm install`
 4. Build the Java modules: `npm run build-plugin`
 5. Build the test report resources: `npm run build-resources`
-> Note: The below steps are only required if you want to debug the Java Language Server plugin
-5. Import `java-extension/com.microsoft.java.test.plugin` in Eclipse
-6. Click `Window` > `Preferences` > `Plug-in Development` > `Target Platform`
-7. In the `Target definitions` panel, select `JDTLS.EXT - /com.microsoft.java.test.plugin/target.target`
-8. Click `Apply and Close`
+6. Install the [Eclipse PDE Support extension](https://marketplace.visualstudio.com/items?itemName=yaozheng.vscode-pde) in your VS Code
+7. Open a Java file and wait until 👍 shows in the right-bottom of the status bar
+    > Note: Step 6 & 7 are only required if you want to debug the code in Java Test Plugin. Sometimes, if you find the code navigation is not working in the Java code, please reload your VS Code.
+
+### Debugging
 
-### Debugging the Extension Client
 1. Open the base directory of Java Test Runner in VS Code
 2. Hit `F5` to launch the extension in debug mode
```

Comment:
Hit `F5` (or run `Launch Extension` in the Debug viewlet) to launch the extension in debug mode